src/cryptonote_basic/miner.cpp

```diff
@@ -835,6 +835,12 @@ namespace cryptonote
 
     #elif defined(__APPLE__)
         
+      if(&IOPSGetTimeRemainingEstimate == NULL)
```

Comment:
Does that really work ? If standard C++, it would not, but there might be custom support in Apple's compiler ?